easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.4-foss-2018a.eb

```diff
@@ -0,0 +1,45 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2017 University of Geneva
+# Authors::   Yann Sagon <yann.sagon@unige.ch>
+# License::   MIT/GPL
+# $Id$
+#
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'MaSuRCA'
+version = '3.2.4'
+
+homepage = 'http://www.genome.umd.edu/masurca.html'
+
+description = '''MaSuRCA is whole genome assembly software. It combines the efficiency of the de Bruijn graph  
+ and Overlap-Layout-Consensus (OLC) approaches. MaSuRCA can assemble data sets containing 
+ only short reads from Illumina sequencing or a mixture of short reads and long reads 
+ (Sanger, 454, Pacbio and Nanopore).'''
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+# need a temporary url to download it. Do it manually here:
+# http://www.genome.umd.edu/masurca_form.html
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['759d5b0411b048d996df1ca6daadf1cc49ff88f4436a21cd81d7f191a8bd80b0']
+
+dependencies = [
+    ('libreadline', '6.3', '', True),
```

Comment:
@ebirn This should be `7.0`, and you shouldn't force the `dummy`-built `libreadline` here?

Reply:
I guess it should, didn't see the v7.0 when I put this together. Generally though: if there is a 'foss' and a 'GCCcore' version of a dependency, the 'GCCcore' would take precedence?

Reply:
By default, no, the `foss` version would be picked up by default, only when that is missing `eb` will consider the `GCCcore` one.

Unless you have `--minimal-toolchains` enabled in your EasyBuild configuration, then it's the reverse (cfr. http://easybuild.readthedocs.io/en/latest/Manipulating_dependencies.html#using-minimal-toolchains-for-dependencies).